packages/account/src/Sections/Security/ConnectedApps/connected-apps.scss

```diff
@@ -0,0 +1,154 @@
+.connected-apps {
+    &__wrapper {
+        display: grid;
+        gap: 1.6rem;
+        grid-template-rows: min-content;
+        grid-template-columns: 1fr min-content;
+        @include mobile {
+            grid-template-columns: 1fr;
+            padding: 1.6rem;
+        }
+    }
+
+    &__content {
+        &--wrapper {
+            display: flex;
+            flex-direction: column;
+            gap: 2.4rem;
+            @include mobile {
+                gap: 1.6rem;
+            }
+        }
+    }
+
+    &__list {
+        &--wrapper {
+            display: flex;
+            flex-direction: column;
+            gap: 0.8rem;
+        }
+
+        &--ordered_list {
+            list-style: auto;
+            font-size: 1rem;
+            padding: 1.5rem;
+            gap: 0.5rem;
+
+            & li {
+                margin-bottom: 0.5rem;
+            }
+        }
+
+        &--row {
+            display: flex;
+            justify-content: space-between;
+            gap: 1.6rem;
+        }
+
+        &--template {
+            background: var(--general-section-1);
+            border-radius: $BORDER_RADIUS * 2;
+            padding: 1.6rem;
+        }
+
+        &--name,
+        &--last-login,
+        &--permission,
+        &--revoke {
+            display: flex;
+            flex-direction: column;
+        }
+
+        &--last-login {
+            width: min-content;
+        }
+
+        &--revoke {
+            justify-content: flex-end;
+        }
+    }
+
+    &__tabular {
+        &--wrapper {
+            height: 28rem;
+        }
+    }
+
+    &__articles {
+        &--wrapper {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            gap: 1.6rem;
+            margin-left: 0.8rem;
+            @include mobile {
+                margin-left: 0;
+            }
+
+            .da-article {
+                margin: 0;
+                @include mobile {
+                    width: 100%;
+                }
+            }
+        }
+    }
+
+    &__empty {
+        &--wrapper {
+            display: flex;
+            flex-direction: column;
+            gap: 0.8rem;
+            margin-top: 9.6rem;
+            @include mobile {
+                gap: 1.6rem;
+                margin-top: 0;
+            }
+        }
+
+        &--bullets {
+            display: flex;
+            flex-direction: column;
+            gap: 0.8rem;
+            @include mobile {
+                gap: 0.4rem;
+            }
+        }
+
+        &--entry {
+            display: flex;
+            gap: 0.4rem;
+        }
+    }
+
+    // Styling for DataTable displayed in Desktop view
+    .table__body .table__row {
+        border-bottom: 1px solid var(--general-section-1);
+    }
+
+    &__table {
+        height: 100%;
+        flex: 1;
+    }
+
+    &__row {
+        grid-template-columns: 169px 198px 80px 104px;
+        padding: 0;
+        column-gap: 42px;
+        text-align: left;
```

Comment:
do we need text-align here? Can we pass this as props to Text component

Reply:
text-align wasn't required. removed it.